src/page/Login.jsx

```diff
@@ -0,0 +1,45 @@
+import PropTypes from "prop-types";
+
+import Input from "../component/form/Input";
+
+import './login.scss';
+
+const Login = ({handleSubmit, onUsernameChange, onPasswordChange, username, password, error}) => {
+
+    return (<>
+        <div className="login__background"></div>
```

Comment:
You can self close an empty tag.
```suggestion
        <div className="login__background" />
```